### PR TITLE
Adding the guestagent.

### PR DESCRIPTION
**Pull request by hub-cap**
- Added the manager code to service
- Updated conf so it works on a guest
- Added exceptions for the guest
- Added all the guest code from legacy reddwarf

### CODE REVIEW

File: bin/reddwarf-guestagent [hunk 1]
```diff
@@ -37,6 +39,9 @@ if os.path.exists(os.path.join(possible_topdir, 'reddwarf', '__init__.py')):
 from reddwarf import version
 from reddwarf.common import config
 from reddwarf.common import service
+# TODO(hub-cap): find out why the db api isint being imported properly
+#from reddwarf.db import db_api
```
Comment (cp16net) [inline bin/reddwarf-guestagent:43]: This might be from a dependency loop like @TimSimpsonR and I saw on Friday.

Comment (hub-cap) [inline bin/reddwarf-guestagent:43]: Naw, i also commented on that dep loop in the PR, i have known about it for a while. Remember when we brought the params into the get_client method in the models? same issue.

**Comment by TimSimpsonR** [comment]
+1

File: reddwarf/guestagent/__init__.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # vim: tabstop=4 shiftwidth=4 softtabstop=4
 
-# Copyright 2011 OpenStack LLC.
+# Copyright (c) 2011 OpenStack, LLC.
```
Comment (ed-) [inline reddwarf/guestagent/__init__.py:3]: 2012

**Comment by ed-** [comment]
+:1: :cow:

**Pull request merged**